phpunit.xml

```diff
@@ -10,22 +10,19 @@
 		<const name="WP_TEST_ACTIVATED_PLUGINS" value="two-factor/two-factor.php" />
 	</php>
 	<testsuites>
-		<testsuite>
+		<testsuite name="All Tests">
 			<directory suffix=".php">tests</directory>
 		</testsuite>
 	</testsuites>
 	<filter>
 		<whitelist processUncoveredFilesFromWhitelist="false">
-			<directory suffix=".php">.</directory>
```

Comment:
This was causing 10 second delay before tests while it parsed all files in the excluded directories.